util/topgen/BUILD

```diff
@@ -13,12 +13,14 @@ py_library(
         "__init__.py",
         "entropy_buffer_generator.py",
         "gen_top_docs.py",
+        "secure_prng.py",
         "strong_random.py",
         "validate.py",
     ],
     deps = [
         ":merge",
         requirement("tabulate"),
+        requirement("pycryptodome"),
```

Comment:
Is this already in the python requirements?

Reply:
yes
`pycryptodome==3.15.0`